src/backend/language.cpp

```diff
@@ -79,238 +80,1049 @@ LanguageMap::LanguageMap() {
-    addLanguage({"amu"}, QT_TRANSLATE_NOOP("QObject", "Amuzgo, Guerrero"));
-    addLanguage({"ang"}, QT_TRANSLATE_NOOP("QObject", "English, Old (ca.450-1100)"));
-    addLanguage({"ar"}, QT_TRANSLATE_NOOP("QObject", "Arabic"));
-    //  addLanguage({"as"}, QT_TRANSLATE_NOOP("QObject", "Assamese"));
-    addLanguage({"az"}, QT_TRANSLATE_NOOP("QObject", "Azerbaijani"));
-    addLanguage({"azb"}, QT_TRANSLATE_NOOP("QObject", "Azerbaijani, South"));
-    //  addLanguage({"ba"}, QT_TRANSLATE_NOOP("QObject", "Bashkir"));
+    addLanguage({"aer"}, QT_TRANSLATE_NOOP("QObject", "Eastern Arrernte"));
+    addLanguage({"aey"}, QT_TRANSLATE_NOOP("QObject", "Amele"));
+    addLanguage({"afr", "af"}, QT_TRANSLATE_NOOP("QObject", "Afrikaans"));
+    addLanguage({"agd"}, QT_TRANSLATE_NOOP("QObject", "Agarabi"));
+    addLanguage({"agg"}, QT_TRANSLATE_NOOP("QObject", "Angor"));
+    addLanguage({"agm"}, QT_TRANSLATE_NOOP("QObject", "Angaataha"));
+    addLanguage({"agn"}, QT_TRANSLATE_NOOP("QObject", "Agutaynen"));
+    addLanguage({"agr"}, QT_TRANSLATE_NOOP("QObject", "Aguaruna"));
+    addLanguage({"agt"}, QT_TRANSLATE_NOOP("QObject", "Central Cagayan Agta"));
+    addLanguage({"agu"}, QT_TRANSLATE_NOOP("QObject", "Aguacateco"));
+    addLanguage({"agx"}, QT_TRANSLATE_NOOP("QObject", "Aghul"));
+    addLanguage({"agx-Cyrl"}, QT_TRANSLATE_NOOP("QObject", "Aghul (Cyrillic script)"));
+    addLanguage({"aia"}, QT_TRANSLATE_NOOP("QObject", "Arosi"));
+    addLanguage({"aii"}, QT_TRANSLATE_NOOP("QObject", "Assyrian Neo-Aramaic"));
+    addLanguage({"ake"}, QT_TRANSLATE_NOOP("QObject", "Akawaio"));
+    addLanguage({"akh"}, QT_TRANSLATE_NOOP("QObject", "Angal Heneng"));
+    addLanguage({"alp"}, QT_TRANSLATE_NOOP("QObject", "Alune"));
+    addLanguage({"alq"}, QT_TRANSLATE_NOOP("QObject", "Algonquin"));
+    addLanguage({"als"}, QT_TRANSLATE_NOOP("QObject", "Tosk Albanian"));
+    addLanguage({"alt"}, QT_TRANSLATE_NOOP("QObject", "Southern Altai"));
+    addLanguage({"alt-Cyrl"}, QT_TRANSLATE_NOOP("QObject", "Southern Altai (Cyrillic script)"));
+    addLanguage({"aly"}, QT_TRANSLATE_NOOP("QObject", "Alyawarr"));
+    addLanguage({"ame"}, QT_TRANSLATE_NOOP("QObject", "Yanesha'"));
+    addLanguage({"amf"}, QT_TRANSLATE_NOOP("QObject", "Hamer-Banna"));
+    addLanguage({"amh", "am"}, QT_TRANSLATE_NOOP("QObject", "Amharic"));
+    addLanguage({"amk"}, QT_TRANSLATE_NOOP("QObject", "Ambai"));
+    addLanguage({"amm"}, QT_TRANSLATE_NOOP("QObject", "Ama (Papua New Guinea)"));
+    addLanguage({"amn"}, QT_TRANSLATE_NOOP("QObject", "Amanab"));
+    addLanguage({"amp"}, QT_TRANSLATE_NOOP("QObject", "Alamblak"));
+    addLanguage({"amr"}, QT_TRANSLATE_NOOP("QObject", "Amarakaeri"));
+    addLanguage({"amu"}, QT_TRANSLATE_NOOP("QObject", "Guerrero Amuzgo"));
+    addLanguage({"amx"}, QT_TRANSLATE_NOOP("QObject", "Anmatyerre"));
+    addLanguage({"ang"}, QT_TRANSLATE_NOOP("QObject", "Old English (ca.450-1100)"));
```

Comment:
We previously had the string "English, Old (ca.450-1100)", because these strings are used a list box in our settings dialog, which sorts them by name. This changes the ordering of those strings, perhaps making it difficult for users to find the correct settings. I'm not sure what would be the best way to approach this issue.